app/src/main/java/ar/com/wolox/android/example/ui/home/news/NewsDataAdapter.kt

```diff
@@ -22,18 +23,34 @@ class NewsDataAdapter(private val newsDataList: ArrayList<News>) : RecyclerView.
     override fun onBindViewHolder(newsHolder: NewsViewHolder, position: Int) {
         newsHolder.run {
             contactName.text = newsDataList[position].title
-            draweeView.setImageURI(Uri.parse(newsDataList[position].picture))
+
+            /**
+             * I put this condition because some news are broken.
+             */
+            if (!newsDataList[position].picture.isNullOrEmpty()) {
+                draweeView.setImageURI(Uri.parse(newsDataList[position].picture)) }
+
             newsDescription.text = newsDataList[position].text
             newsTime.text = newsDataList[position].readableCreationTime
+
+            onItemClicked(newsDataList[position])
         }
     }
 
     override fun getItemCount(): Int = newsDataList.size
 
     class NewsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
-        val contactName = itemView.news_name
-        val newsDescription = itemView.news_description
+        val contactName = itemView.news_name!!
```

Comment:
Is the not null assertion necessary?

Reply:
It was to solve some "unchecked nullability issues" warnings.